packages/contracts/src/eip712/signatureParamTypes.ts

```diff
@@ -73,3 +73,10 @@ export interface EIP712LinkSignerParams extends Subaccount {
   signer: string;
   nonce: string;
 }
+
+export interface EIP712TransferQuoteParams extends Subaccount {
```

Comment:
So the EIP params should mirror that of what the signature / contract expects, but we can add this "recipientName" abstraction at the engineClient or top-level client

Thoughts on `toSubaccountName` or `recipientSubaccountName` as the property name?

Reply:
i think `recipientSubaccountName` looks good

Reply:
oh i took this pattern from what was done for `EIP712LiquidateSubaccountParams` seemed cleanest to me?

Reply:
Ah that's true 🤔 okay 👍🏻